syntaxes/v.tmLanguage.json

```diff
@@ -1,11 +1,9 @@
 {
 	"name": "V",
-	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
-	"fileTypes": [".v", ".vh", ".vsh"],
```

Comment:
I have this file hard-linked locally so changes to `syntaxes/v.tmLanguage.json` will automatically affect an [Atom package](https://github.com/Cutlery-Drawer/language-v/blob/master/grammars/v.tmLanguage.json) ported from this repository.

Since Atom actually depends on a `fileTypes` array being included in a grammar, it'd be nice if this field wasn't removed.